Library/Formula/cairomm.rb

```diff
@@ -17,21 +14,59 @@ class Cairomm < Formula
 
   deprecated_option "without-x" => "without-x11"
 
-  depends_on 'pkg-config' => :build
+  depends_on "pkg-config" => :build
   if build.cxx11?
-    depends_on 'libsigc++' => 'c++11'
+    depends_on "libsigc++" => "c++11"
   else
-    depends_on 'libsigc++'
+    depends_on "libsigc++"
   end
 
-  depends_on 'cairo'
-  depends_on 'libpng'
+  depends_on "cairo"
+  depends_on "libpng"
   depends_on :x11 => :recommended
 
   def install
     ENV.cxx11 if build.cxx11?
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cc").write <<-EOS.undent
+      #include <string>
+      #include <iostream>
+      #include <cmath>
```

Comment:
Nope.  I thought it was failing to build without them but it was an unrelated error.